main.js

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable no-unused-vars */
+
+var product = [
+  { image: 'images/photo1.jpg',
```

Comment:
These names should either be descriptions of the images or have a hyphen between image and the number.